distrobox: add page (#7638)
tldr-pages/tldr: pages/linux/distrobox-list.md

# distrobox-list

> Manual page viewer for distrobox.

- List all distrobox containers:

`distrobox-list`

> - List all distrobox containers (verbose):

Reviewer: ```suggestion
- List all distrobox containers with verbose information:
```

# distrobox-list

> Manual page viewer for distrobox.

> > More information: <https://distrobox.privatedns.org>

Reviewer: ```suggestion
> More information: <https://distrobox.privatedns.org>.
```

# distrobox-list

> > This command lists available distroboxes. It detects them and lists them separately from the rest of normal podman or docker containers.

Reviewer: ```suggestion
> List available Distrobox containers. It detects them and lists them separately from the rest of normal podman or Docker containers.
```
